### DISCUSSION TITLE
[4.1] [Media Manager] Add fully functional gallery options for images in media manager

### DISCUSSION THREAD

**Discussion by Bond97** [joomla/joomla-cms#47136] [2. Cold Features]
### Is your feature request related to a problem? Please describe.
When I open any image including default on folders `banners` or `headers` there is no toggles that allow you to change image from 1.jpg to 2.jpg without pressing esc key and select another image.
![изображение](https://user-images.githubusercontent.com/78050514/117506354-21125b00-af8e-11eb-8032-caa5ccc8bf72.png)

### Describe the solution you'd like

Add responsive left and right toggles with following function: change image with mouse click or pressing corresponding keys on your keyboard device.

### Additional context

Maybe this question is discussed earlier, but I didn't find it with keywords.

**Comment by brianteeman** [comment]
@bembelimen am i right in thinking there is a 4.1 project for media manager enhancements?

**Comment by brianteeman** [comment]
Can this be tagged 4.1 please
